Bottom: a576465dcf83dcc872db62e03f233b37c691e4fe
Top:    79b4f844cc4303926f52d22394429480bb06a76c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-18 17:38:57 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index d387f43..0b03204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
